pom.xml

```diff
@@ -0,0 +1,281 @@
+                The software is public property and you therefore have the right to the source code, if desired.
+                You may charge fees for distribution, warranties, and services provided in connection with the software or derivative works thereof. The name USGS can be used in any advertising or publicity to endorse or promote any products or commercial entity using this software if specific written permission is obtained from the USGS.
+                The user agrees to appropriately acknowledge the authors and the USGS in publications that result from the use of this software or in products that include this software in whole or in part.
+                Because the software and related material are free (other than nominal materials and handling fees) and provided "as is," the authors, the USGS, and the United States Government have made no warranty, express or implied, as to accuracy or completeness and are not obligated to provide the user with any support, consulting, training or assistance of any kind with regard to the use, operation, and performance of this software nor to provide the user with any updates, revisions, new versions or "bug fixes".
+                The user assumes all risk for any damages whatsoever resulting from loss of use, data, or profits arising in connection with the access, use, quality, or performance of this software.
+            </comments>
+        </license>
+    </licenses>
+
+    <scm>
+        <connection>scm:git:https://github.com/usgs/aqts-capture-dvstat-transform.git</connection>
+        <developerConnection>scm:git:git@github.com:usgs/aqts-capture-dvstat-transform.git</developerConnection>
+        <url>https://github.com/usgs/aqts-capture-dvstat-transform</url>
+        <tag>HEAD</tag>
+    </scm>
+
+    <repositories>
+        <repository>
+            <id>Central</id>
+            <name>Maven Central</name>
+            <url>https://repo.maven.apache.org/maven2/</url>
+        </repository>
+        <repository>
+            <id>WMA</id>
+            <name>WMA Artifactory</name>
+            <url>${wma.maven.url}/list/wma-maven-centralized/</url>
+        </repository>
+    </repositories>
+
+    <distributionManagement>
+        <repository>
+            <id>wma-maven-releases</id>
+            <name>wma-maven-releases</name>
+            <url>${wma.maven.url}/wma-maven-releases</url>
+        </repository>
+        <snapshotRepository>
+            <id>wma-maven-snapshots</id>
+            <name>wma-maven-snapshots</name>
+            <url>${wma.maven.url}/wma-maven-snapshots</url>
+        </snapshotRepository>
+    </distributionManagement>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.2.4.RELEASE</version>
```

Comment:
2.2.5.RELEASE is out but I couldn't get it to run successfully with aqts-capture-ts-description... I think it has something to do with the spring @ComponentScan annotation that's build into the @SpringBootApplication annotation but haven't spent any real time on it yet.